charts/icm-as/system-conf/cluster/empty-config.xml

```diff
@@ -0,0 +1,3 @@
+<!--
```

Comment:
Why is this file checked in?

Reply:
the file serves as a placeholder, I removed the logback-main.xml